index.html

```diff
@@ -11,7 +11,7 @@
     <div class="p-5">
         <h1>Lista de asistencia</h1>
         <ul class="list-group my-4">
-            <li class="list-group-item list-group-item-dark">{ Tu nombre aquí }</li>
+            <li class="list-group-item list-group-item-dark">Juan Meza <div> div></li>
```

Comment:
Hola, tu codigo esta mal, XD arreglalo.

```suggestion
            <li class="list-group-item list-group-item-dark">Juan Meza <div><div></li>
```

Reply:
Listo.